install.sh

```diff
@@ -29,14 +29,12 @@ if [ "$(id -u)" != "0" ]; then
     [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
 fi
 
-# Split path to this install script into an array.
-IFS='/' SRC_PATH=($0)
-# Remove this file name.
-unset SRC_PATH[-1]
+#Create opt folder under root
+mkdir -p /opt
+#Get current shell script directory name
+SRC_PATH=$(cd "$(dirname "$0")"; pwd)
```

Comment:
I made a PR to use `ci_edit` (or other name of current directory) instead of `.` as the install directory since variable `$0` will have the relative path of `./` with normal install script execution.